### ISSUE TITLE
Minor typo-like error in decoding/tests/test_searchlight.py

### ISSUE THREAD

**Issue by SylvainTakerkart** [nilearn/nilearn#1998]
Even if it has no influence on the validity of the test

```
    cond = np.arange(frames, dtype=int) > (frames // 2)
```

...should probably be...

```
    cond = np.arange(frames, dtype=int) >= (frames // 2)
```

on line 25...

**Comment by GaelVaroquaux** [comment]
Tell me why.

**Comment by GaelVaroquaux** [comment]
Can you do a PR to fix that.

**Comment by SylvainTakerkart** [comment]
#2027

**Comment by NicolasGensollen** [comment]
Solved in #2027
